compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -557,6 +584,101 @@ struct QuantizeWeights final : public luci::CircleNodeMutableVisitor<bool>
   }
 };
 
+/**
+ * @brief Quantize const input tensors using min/max of const values
+ */
+void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
+{
+  auto opcode = node->opcode();
+  auto arity = node->arity();
+
+  loco::Node *input_node{nullptr};
+  luci::CircleConst *const_node{nullptr};
+
+  switch (opcode)
+  {
+    case luci::CircleOpcode::CONV_2D:
+    case luci::CircleOpcode::DEPTHWISE_CONV_2D:
+    case luci::CircleOpcode::FULLY_CONNECTED:
+    case luci::CircleOpcode::TRANSPOSE_CONV:
+      // Handled in QuantizeWeights and QuantizeBias
```

Comment:
Do the weights and the bias differ significantly that they are not handled here?

Reply:
Well, I think so. We have to consider something more to quantize them (e.g., granularity (lwq, cwq), weight clipping (maybe implemented in the future), different data type and quantization timing for bias (int32)).